### ISSUE TITLE
Public-facing marketing pages. VoiceOver Chrome not reading Alt Text aloud

### ISSUE THREAD

**Issue by yaelberger-commits** [cds-snc/notification-planning#2034]
## VoiceOver Chrome, Alt text for images not being read. Says instead “object replacement character”

Describe what is failing
- include what is failing
- include an image
- include context

## Potential fix

- We would like to test this with Fable participants to better understand the issue and behaviour

## Resources

- Links to documentation or articles that could be helpful
- Bug bash document where this issue was discovered while QAing public-facing marketing pages https://docs.google.com/document/d/1X7jSElQL1p3-tCJZPbsuX9UfquLK0YiUSoYMJhzxkxc/edit?tab=t.0#task=Pp2yfaXCepaDfKCs

## Acceptance Criteria

- [ ] Should this issue be added to the A11y Statement if it cannot be resolved soon?

**Comment by connorscarolyns** [comment]
Did not resurface in fable testing, closing for now. Will reopen if it arises again.
